### ISSUE TITLE
Build scene data error

### ISSUE THREAD

**Issue by ismailgamedev** [FlaxEngine/FlaxEngine#585]
**Flax 1.2 crashes after Build scene data** 

[Log.txt](https://github.com/FlaxEngine/FlaxEngine/files/6950042/Log.txt)

**Comment by mafiesto4** [comment]
Relevant log part:
```
[Fatal] Assertion failed!
File: c:\flax\flaxengine\source\engine\content\storage\flaxstorage.cpp
Line: 1295

Expression: _chunksLock == 0
```

**Comment by mafiesto4** [comment]
Probably related to texture streaming or something in conjunction to:
`Updating Env probe 'EnvironmentProbe (FlaxEngine.EnvironmentProbe; 91bf2ec947e9ba1a23f917a1e563ff05)'...`
So my guess is that texture streaming is reading from file when we update the probe. Relevant code location to work on debug/fix it: `EnvironmentProbe::SetProbeData`

**Comment by mafiesto4** [comment]
Fixed in 3dbff3cab795164aa8bb9b392d16a92c83c60acc
